chapter06/docker-compose.yml

```diff
@@ -64,25 +63,13 @@ x-airflow-common:
     AIRFLOW__API__AUTH_BACKENDS: 'airflow.api.auth.backend.basic_auth,airflow.api.auth.backend.session'
     AIRFLOW__SCHEDULER__DAG_DIR_LIST_INTERVAL : 60
     AIRFLOW__CORE__TEST_CONNECTION: 'Enabled'
-
```

Comment:
Why is this removed. I would like to keep the docker compose files for all chapters as much aligned as possible. Also as much as possible equal to the official docker compose file. I think I only changed minor things like the data volume and the disabling of the examples.

Reply:
i agree with you. I will bring it back closest to the latest docker compose version